todoapp/users/serializers.py

```
@@ -6,5 +6,29 @@
 
+from users import models as users_models
+
 
 # Add your serializers
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = users_models.CustomUser
+        fields = ["id", "first_name", "last_name", "email", "date_joined"]
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)